jcheck/src/main/java/org/openjdk/skara/jcheck/IssuesTitleCheck.java

```diff
@@ -88,11 +88,6 @@ private boolean hasTrailingPeriod(String description) {
     }
 
     private boolean hasLeadingLowerCaseLetter(String description) {
-        if (Character.isUpperCase(description.charAt(0))) {
-            return false;
-        }
-        var firstWord = description.split(" ")[0];
-        // If first word contains special character, it's very likely a reference to file or function, ignore it
-        return !FILE_OR_FUNCTION_PATTERN.matcher(firstWord).matches();
+        return ALL_LOWER_CASE_PATTERN.matcher(description.split(" ")[0]).matches();
```

Comment:
This could all be captured in the regex instead of splitting the string first. Something like `"[a-z]+(:?\\h.*)?"` should do it I think.

Reply:
Is the `:?` needed? It seems redundant, but I'm not a regex expert.

Reply:
I think Erik has a typo here, should be `"[a-z]+(?:\\h.*)?"`?

Reply:
On closer look, the "`:?`" won't do what we want. It will match one of the patterns we explicitly do not want to match: `"abc: def"`. So I think that should just be: `"[a-z]+(\\h.*)?"`

Reply:
I think Erik was trying to use the non-capturing pattern `?:...`

Reply:
Ah, you may be right (in which case _that_ would be fine).

Reply:
That was indeed what I was going for.